in_toto/verifylib.py

```diff
@@ -1306,6 +1332,53 @@ def verify_threshold_constraints(layout, chain_link_dict):
                     step_name=step.name, keyid=keyid)))
 
 
+def print_error_message(rule, matched_artifacts, rule_trace, error_traceback):
```

Comment:
Please be mindful when you choose function names. Judging from the name this looks like a generic function to print error messages, but it is neither generic nor does it print something (see suggestion above for an alternative name).